browser/js/reducers/assessmentTeam.js

```diff
@@ -0,0 +1,34 @@
+'use strict'
+
+import {
+  LOAD_TEAM_REQUEST,
+  LOAD_TEAM_SUCCESS,
+  LOAD_TEAM_FAILURE
+} from '../actions/assessmentTeamActions'
+
+const initialState = {
+  teamFetching: false,
+  error: null
+}
+
+export default function assessmentTeam (state = initialState, action) {
+  console.log('assessmentTeam.js', action)
+  switch (action.type) {
+    case LOAD_TEAM_REQUEST:
+      return Object.assign({}, state, {
+        teamFetching: true
+      })
+    case LOAD_TEAM_SUCCESS:
+      return Object.assign({}, state, {
+        teamFetching: false,
+        team: action.team
+      })
+    case LOAD_TEAM_FAILURE:
```

Comment:
Not sure if this would be the right spot for it, but you might consider implementing a simple error displaying/reporting system for errors that are important to the end user. For example there's [this library](https://github.com/Moosylvania/react-growl) which seems cool (full disclosure: I've used stuff like this, but not this exact library).